### PR TITLE
ADAPT-5212 | @rebeccahongsf | Add trip dates for Notify Me forms

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Add trip dates for Notify Me forms

# Review Tasks

## Setup tasks and/or behavior to test

1. Login as a test user
2. Navigate to [`/travel-study/ts-test/italy-notify-me-form/`](https://deploy-preview-416--stanford-alumni.netlify.app/travel-study/ts-test/italy-notify-me-form/)
3. Verify that the `su_trip_start_date` and `su_trip_end_date` are prefilling

# Associated Issues and/or People
- ADAPT-5212

### CODE REVIEW

**Comment by sherakama** [comment]
Screen Shot 2022-07-27 at 4 23 42 PM

500 error on the form script?

**Comment by rebeccahongsf** [comment]
> Screen Shot 2022-07-27 at 4 23 42 PM
> 
> 500 error on the form script?

Haha I was just looking into this @sherakama. Looks like the [preview](https://preview.kimbia.com/preview/travel-study-public/public-notify-me) isn't working either. Could it be because of my `su_trip_start_date` and `su_trip_end_date` ADC value addition? 🤔

**Comment by rebeccahongsf** [comment]
I figured it out — accidentally added a blank ADC value. I removed it and now all should be working as expected now @sherakama 👍

**Comment by rebeccahongsf** [comment]
image
 
Looks to be working as expected 👀 I'm going to include the time trim since I'm already in there. Will be pushing up a fix for that in a bit.

**Comment by sherakama** [comment]
The form 500s on a blank ADC value! Yikes.

**Comment by rebeccahongsf** [comment]
> The form 500s on a blank ADC value! Yikes.

Right? There should be a _field required_ check when adding those ADC values 😬

**Review by sherakama — APPROVED** [review]

File: src/components/page-types/formPage/formPage.js [hunk 1]
```diff
@@ -58,6 +57,18 @@ const FormPage = (props) => {
...
+
+    // Trip Dates for Notify Me form
+    window.su_trip_url = trip?.full_slug;
+    if (trip?.content?.startDate && trip?.content?.endDate) {
+      window.su_trip_start_date = new Date(trip?.content?.startDate)
+        ?.toISOString()
+        ?.split('T', 1)[0];
+      window.su_trip_end_date = new Date(trip?.content?.endDate)
+        ?.toISOString()
+        ?.split('T', 1)[0];
```
Comment (rebeccahongsf) [inline src/components/page-types/formPage/formPage.js:69]: I merged this PR in to unblock Robert's testing but I'd love for you to take a glance and let me know your thoughts on this implementation for Date time trim when you have the chance @sherakama 😃

**Pull request merged**